tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature

```diff
@@ -458,7 +458,7 @@ Feature: sharing
       | uid_owner   | %username%     |
     Then as "Brian" file "/Shares/textfile0.txt" should exist
     And as "Carol" file "/Shares/textfile0.txt" should exist
-    When the administrator deletes group "grp1" using the provisioning API
+    When the administrator deletes group "grp1" from the default user backend
```

Comment:
@nabim777 this step calls `$this->deleteGroup($group);` function , which now only deletes the `ldap` group so this won't work in ocis is that the expected behaviour?

Reply:
@SwikritiT at the top line:438 there is `# deleting an LDAP group is not relevant or possible using the provisioning API` so I used ldapDelete instead of the provision. Should I supposed to delete this scenario since there is ocs endpoint removed.

Reply:
these tests are skipped in reva look at the top of the file it has `@skipOnReva` tag and as far as I understand the comment is probably the reason why the test is skipped in reva becasue deleting group in ldap is not possible by our `ocs` api. You can change this scenario to use graph api if possible, look at the scenario and try to understand what it is doing. If you think we already have such a test cover by our graph api you can remove it or try to make it work wth graph api. But the scenario is tagged with issue `2441` so might the test might be in expected to fail file

Reply:
If the ocs endpoint is removed then `using ocs API version....` and checking `ocs status code` are not relevant. The scenario is skipped in CI, so test locally and check if the issue is fixed with graph api.